reference-architectures/reference-architecture-ccs.md

```diff
@@ -0,0 +1,157 @@
+---
+
+copyright:
+  years: 2024
+lastupdated: "2024-06-12"
+
+subcollection: security-services
+
+authors:
+- name: Dharmesh Bhakta
+  email: bhakta@ibm.com
+
+# The release that the reference architecture describes
+version: 1.0
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: <url>
+
+use-case:
+  - CloudSecurity
+  - KeyManagement
+  - CloudSecurityAndCompliance
+  - DataCompliance
+  - Governance
+  - GRC
+
+industry: SoftwareAndPlatformApplications, Technology, Banking, FinancialSector
+
+compliance: CIS Benchmarks
+
+docs: https://cloud.ibm.com/docs/<subsection>
+
+content-type: reference-architecture
+
+production: false
+
+---
+<!--
+The following line inserts all the attribute definitions. Don't delete.
+-->
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# Core Security Services Stack on IBM Cloud
```

Comment:
```suggestion
# Core security services
```

we don't want to refer to `stack` and this will be viewed on IBM Cloud. 

I assume that it's sentence capitalization because it's not a product.